lib/request-processor.js

```diff
@@ -2,7 +2,7 @@ const request = require('request')
 const os = require('os')
 const uuidv4 = require('uuid/v4')
 const logger = require('./logger')
-
+const assert = require('assert')
```

Comment:
nit: ordering/spacing.

should come before `logger`, should have a newline after require calls